src/test/java/org/opensearch/neuralsearch/processor/normalization/RRFNormalizationTechniqueTests.java

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.neuralsearch.processor.normalization;
+
+import org.apache.lucene.search.ScoreDoc;
+import org.apache.lucene.search.TopDocs;
+import org.apache.lucene.search.TotalHits;
+import org.opensearch.neuralsearch.processor.CompoundTopDocs;
+import org.opensearch.neuralsearch.processor.NormalizeScoresDTO;
+import org.opensearch.neuralsearch.query.OpenSearchQueryTestCase;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Abstracts testing of normalization of scores based on RRF method
+ */
+public class RRFNormalizationTechniqueTests extends OpenSearchQueryTestCase {
+    private static final float DELTA_FOR_ASSERTION = 0.001f;
+    static final int RANK_CONSTANT = 60;
```

Comment:
You can import it from `RRFNormalizationTechnique`.

Reply:
those are two different semantics, I don't want to mix them. Besides, it's a coincident that both have same value. I can create one for general assertion somewhere in test util classes

Reply:
moved constant of 0.001 to a base class

Reply:
> those are two different semantics, I don't want to mix them. Besides, it's a coincident that both have same value. I can create one for general assertion somewhere in test util classes

Makes sense.